hashicorp/boundary-ui
# chore: 🤖 udate hds from v3.5 to v4

# Description
closes: https://hashicorp.atlassian.net/browse/ICU-14064 

Resolution added to package.json, [slack msg](https://hashicorp.slack.com/archives/C7KTUHNUS/p1718644636513669) for details.

Check out the hds [upgrade release guide](https://github.com/hashicorp/design-system/releases/tag/%40hashicorp%2Fdesign-system-components%404.0.0) for details.

## How to Test

1. navigate to ui/admin
2. run yarn start
3. verify ui is rendering with no errors in console and no styles missing
4. repeat steps for ui/desktop

## Checklist
